src/Dfe.PlanTech.Infrastructure.Contentful/Helpers/ContentfulSetup.cs

```diff
@@ -67,3 +74,17 @@ public static IServiceCollection SetupRichTextRenderer(this IServiceCollection s
     private static Func<Type, bool> IsContentRenderer(Type contentRendererType)
         => type => type.IsClass && !type.IsAbstract && type.IsSubclassOf(contentRendererType);
 }
+
+public class CustomDelegatingHandlerTokenRefresher : DelegatingHandler
```

Comment:
Remove this class

Reply:
[a7ee237](https://github.com/DFE-Digital/sts-plan-technology-for-your-school/pull/900/commits/a7ee23746b4af6efd015645300904b889a4e35c8)